pkg/scheduler/framework/runtime/framework.go

```diff
@@ -737,10 +738,14 @@ func (f *frameworkImpl) RunPreFilterPlugins(ctx context.Context, state *framewor
 			}
 
 			// When PreFilterResult filters out Nodes, the framework considers Nodes that are filtered out as getting "UnschedulableAndUnresolvable".
-			return result, framework.NewStatus(framework.UnschedulableAndUnresolvable, msg)
+			returnStatus = framework.NewStatus(framework.UnschedulableAndUnresolvable, msg)
```

Comment:
Sorry, I may forget a past conversation; Do we actually have to change here to continue to run all plugins? 

Let's say PreFilterResult(s) from pluginA and pluginB filtered out all nodes, this Pod will never be schedulable until pluginA or pluginB changes its scheduling results, right? 
In other words, even if we continued to run pluginC etc, and let's say pluginC also returned PreFilterResult, it wouldn't matter to decide when to retry the Pod; even if pluginC changes its scheduling result, the Pod won't be schedulable until pluginA and pluginB does; meaning, pluginC isn't worthwhile to put into `diagnosis.UnschedulablePlugins`.

Reply:
> Sorry, I may forget a past conversation; Do we actually have to change here to continue to run all plugins?
> 
> Let's say PreFilterResult(s) from pluginA and pluginB filtered out all nodes, this Pod will never be schedulable until pluginA or pluginB changes its scheduling results, right? In other words, even if we continued to run pluginC etc, and let's say pluginC also returned PreFilterResult, it wouldn't matter to decide when to retry the Pod; even if pluginC changes its scheduling result, the Pod won't be schedulable until pluginA and pluginB does; meaning, pluginC isn't worthwhile to put into `diagnosis.UnschedulablePlugins`.

the past conversation is this [comment](https://github.com/kubernetes/kubernetes/pull/122251#discussion_r1636205128). don't run all plugins because `break` In L741.

Reply:
Oh, how I missed `break` 😅 

But, instead of `break` can we do this if you take my suggestion?

```go
return result, framework.NewStatus(framework.UnschedulableAndUnresolvable, msg), pluginsWithNodes
```

Reply:
done